### ISSUE TITLE
Custom Pause/Play Shorcut does not with Quick Nav enbaled

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#331]
Moved from #327 ...
In today's interview with our VoiceOver Consultant we learned that the custom Pause/Play shortcut does not work with Quick Nav enabled. It works fine and as designed when Quick Nav is not on.

@jessegreenberg, I am assuming that's just how it is. Do you have any comment about this?

**Comment by jessegreenberg** [comment]
I just tested Safari with VoiceOver on macOS Catalina 10.15.4 and I was able to use the shortcut with Quick Nav enabled. Sorry, I am not sure what was happening during the interview.

**Comment by terracoda** [comment]
I'll test it tomorrow. It was not working in the interview session until Quick Nav was turned off.

**Comment by terracoda** [comment]
Works fine for me, too, with Quick Nav enabled. Not sure what the issues was. Closing and we'll see if it comes up in RC testing.
